src/web/app/pages-admin/admin-notifications-page/admin-notifications-page-data.ts

```diff
@@ -0,0 +1,55 @@
+import moment from 'moment-timezone';
+import { Notification, NotificationStyle, NotificationTargetUser } from '../../../types/api-output';
+import { NotificationEditFormModel } from './notification-edit-form/notification-edit-form-model';
+
+/**
+ * Structure for example of notification edit model.
+ */
+export const EXAMPLE_NOTIFICATION_EDIT_MODEL: NotificationEditFormModel = {
+  notificationId: 'notification1',
+  shown: false,
+
+  startTime: { hour: 0, minute: 0 },
+  startDate: { year: 0, month: 0, day: 0 },
+  endTime: { hour: 0, minute: 0 },
+  endDate: { year: 0, month: 0, day: 0 },
+
+  style: NotificationStyle.SUCCESS,
+  targetUser: NotificationTargetUser.INSTRUCTOR,
+
+  title: 'valid title',
+  message: 'valid message',
+
+  isSaving: false,
+  isDeleting: false,
+};
+/**
+ * Structure for example of notification.
+ */
+export const EXAMPLE_NOTIFICATION_ONE: Notification = {
+  notificationId: 'notification1',
+  startTimestamp: moment('2017-09-15 09:30:00').valueOf(),
```

Comment:
Since the two example notifications have dates that differ only in year, the correct sorting of notifications is not apparent for the notification table snapshots since the year is not displayed in the UI. I would suggest making the dates here more varied so that the correct behaviour is more apparent from looking at the snapshots.